zowe-rest-api-commons-spring/src/main/java/org/zowe/commons/zos/security/platform/PlatformErrno2.java

```diff
@@ -0,0 +1,1807 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.commons.zos.security.platform;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.Generated;
+
+/**
+ * Provides explanation for secondary error codes for as described at:
+ * https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.bpxa800/errnojrs.htm
+ */
+@Generated
+public enum PlatformErrno2 {
```

Comment:
this seems useful to have these - i suppose in the future if IBM adds to their list - we'll just open PRs to update this as needed?

Reply:
This has been generated from the documentation - https://gist.github.com/plavjanik/a0f50e61edc2e2419a161e8d736e86de. Do you think that we should include the script into the source code?

Reply:
ahh - that's a cool script!  that might be neat to include if its not much trouble or maybe just a link to your gist